fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionScheduler.java

```diff
@@ -353,6 +356,7 @@ private List<CompactionTask> createCompactionTasks(long currentVersion, Map<Long
             request.timeoutMs = LakeService.TIMEOUT_COMPACT;
             request.allowPartialSuccess = allowPartialSuccess;
             request.encryptionMeta = GlobalStateMgr.getCurrentState().getKeyMgr().getCurrentKEKAsEncryptionMeta();
+            request.forceBaseCompaction = (priority == PartitionStatistics.CompactionPriority.MANUAL_COMPACT);
```

Comment:
how to distinguish between `cumulative compact` and `base compact`?

Reply:
native uses `is_base_compaction` to distinguish, if set to false, it will be cumulative compaction, so the behavior is the same.

Reply:
but cloud table do not support cumulative compaction command, so it can be done later, right now, this is enough